Sources/StringsLintFramework/Rules/Lint/JSONCommentRule.swift

```diff
@@ -117,6 +124,15 @@ extension JSONCommentRule: LintRule {
         return .invalidJSON
     }
 
+      do {
+          let jsonDict = try JSONSerialization.jsonObject(with: jsonData, options: []) as! [String: Any]
+          for (k, _) in jsonDict {
```

Comment:
Could probably optimize this a bit. Something like this maybe?
```suggestion
let invalidKeys = Set(jsonDict.map { $0.key } ).subtracting(validKeys)
if invalidKeys.isNotEmpty {
    return .invalidJSONKey(invalidKey: invalidKeys)
}
```

The `invalidJSONKey` also should probably take an array of keys not just a single key

Reply:
yah make sense